frontend/src/DashboardView/components/TeamList/TeamList.js

```diff
@@ -97,12 +97,23 @@ class TeamList extends React.Component {
 				</Mutation>
 				<h1>My Teams</h1>
 				<style.TeamsList>
-					<Query query={query.FIND_TEAMS_BY_USER}>
-						{({ loading, error, data: { findTeamsByUser } }) => {
+					<Query query={query.FIND_TEAMS_BY_USER} notifyOnNetworkStatusChange>
+						{({
+							loading,
+							error,
+							data: { findTeamsByUser },
+							refetch,
+							networkStatus
```

Comment:
what is networkStatus? it is a part of the graphQL query that keeps track of network status!